src/hotkey/hotkey_item.cpp

```diff
@@ -157,6 +157,7 @@ bool hotkey_base::matches(const SDL_Event &event) const
 void hotkey_base::save(config& item) const
 {
 	item["command"] = get_command();
+	item["disabled"] = is_disabled() ? config::attribute_value::s_true : config::attribute_value::s_false;
```

Comment:
Okay, I didn't realise the hot-key load/save functions are able to resolve booleans on their own. I'll remove the config attributes shortly.

I know it's not always easy to help newbies like me, but I find comments like 'you should use x instead of y because this is how the system works' to be much more helpful than just 'don't do this'.

Thanks.